lib/ocpp/v201/ctrlr_component_variables.cpp

```diff
@@ -1116,6 +1116,20 @@ const RequiredComponentVariable& SecurityProfile = {
         "SecurityProfile",
     }),
 };
+const ComponentVariable& AllowCSMSRootCertificateInstallWhenLowSecurityProfile = {
+    ControllerComponents::SecurityCtrlr,
```

Comment:
They should be part of the InternalCtrlr since they are configuration items used internally

Reply:
done